azure-pipelines.yml

```diff
@@ -44,16 +50,25 @@ steps:
     condition: not(contains(variables['BHCommitMessage'],'!skipcodecoverage'))
   
   - task: ArchiveFiles@2
-    displayName: 'ZIP up build directory'
+    displayName: 'ZIP build directory'
```

Comment:
Is there a reason to zip up the build directory before you publish it?  You can publish an artifact that is multiple files so there's not really a good reason to do this?

Reply:
Yep, it's so you can grab the zip of files if you need to check/test something from the actual build artifact.  The nupkg will get pushed to the feeds separately but this is mostly for checking on issues that occur during the actual build process.